src/vuestic-theme/vuestic-components/va-count-badge/VaCountBadge.demo.vue

```diff
@@ -1,43 +1,43 @@
 <template>
   <VbDemo>
-    <VbCard title="0">
-      <va-count-badge :number="0">Statuses</va-count-badge>
+    <VbCard width="150px" title="0">
```

Comment:
Will it be possible to add a css class:
```
.vb-card {
  width: 150px;
}
```

Reply:
@jean-moldovan I'm not following.

What's your intention?

Reply:
Remove `width="150px"` from the template to make it a bit cleaner. Add css:
```
.vb-card {
  width: 150px;
}
```

But no problems if not.

Reply:
@jean-moldovan it's a normal situation in demo to have cards with different sizes.
Futhermore, using css classes is discouraged in demo, inline styles are preferred.